### ISSUE TITLE
Missing symbols on Binance

### ISSUE THREAD

**Issue by LukasUlrych** [ccxt/ccxt#8366]
Hi there,
there seems to be missing symbols on Binance, specifically running `[s for s in exchange.symbols if 'BTC' in s]` in python returns only `['BTC/BUSD', 'BTC/USDT']`. The same holds for `[s for s in exchange.markets.keys() if 'BTC' in s]`. Other exchanges seem to be ok.

ccxt version: 1.41.19
python: 3.8.7
os: win 10

**Comment by kroitor** [comment]
@LukasUlrych hi! Which specific market is missing?

**Comment by LukasUlrych** [comment]
Well, `ADA/BTC`, `ETH/BTC`, `LTC/BTC` etc., pretty much any altcoin vs. BTC. I can see only USDT or BUSD pairs.

**Comment by LukasUlrych** [comment]
@kroitor I am not sure whether this is only local issue or not. I tried going back to version 1.41.10 and the symbols are all there. I noticed this after update today.

**Comment by kroitor** [comment]
@LukasUlrych always use the most recent version and make sure it's the most recent number by printing it in runtime (don't trust the pip output). Let me know if this does not help:

```Python
import ccxt
from pprint import pprint
print('CCXT Version:', ccxt.__version__)
exchange = ccxt.binance({
    'enableRateLimit': True,
    'options': {'defaultType':'spot'},  # spot is the default
})
markets = exchange.load_markets()
pprint(exchange.symbols)
```

```Python
import ccxt
from pprint import pprint
print('CCXT Version:', ccxt.__version__)
exchange = ccxt.binance({
    'enableRateLimit': True,
    'options': {'defaultType':'future'},
})
markets = exchange.load_markets()
pprint(exchange.symbols)
```

```Python
import ccxt
from pprint import pprint
print('CCXT Version:', ccxt.__version__)
exchange = ccxt.binance({
    'enableRateLimit': True,
    'options': {'defaultType':'delivery'},
})
markets = exchange.load_markets()
pprint(exchange.symbols)
```

**Comment by LukasUlrych** [comment]
@kroitor Thank you, it helped! The issue seems to be that running just 
```
exchange = ccxt.binance({
    'enableRateLimit': True
})
```
defaults to defaultType = 'future' not defaultType = 'spot'.

**Comment by kroitor** [comment]
@LukasUlrych ah, i see, hold, will fix that in 1.41.20.
